### PR TITLE
[Tabs][updatejsonsection][1360657]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Added properties. Please review.

### CODE REVIEW

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 

"teamType": "The type of the team",
KP: Correct usage of article.

"frameContext": "The context where tab url is loaded (content, task, setting, remove, sidePanel)",
"isPSTNCallingAllowed": "Represents whether PSTN calling is allowed for the current logged in User",
"isCallingAllowed": "Represents whether calling is allowed for the current logged in User",
KP: Correct the capitalization in the above sentences.

Meeting Id
KP: Meeting ID

**Comment by Mahesh2516** [comment]
@KirtiPereira I have made the necessary changes. Please review.

**Review by KirtiPereira — DISMISSED** [review]
Hi @surbhigupta12 - please approve and merge the PR.
Thank you.

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor comments. Please verify.

File: msteams-platform/tabs/how-to/access-teams-context.md [hunk 1]
```diff
@@ -73,20 +73,37 @@ The context variable will look like the following example.
...
+    "teamType": "The type of team",
+    "teamSiteUrl": "The root SharePoint site associated with the team",
+    "teamSiteDomain": "The domain of the root SharePoint site associated with the team",
+    "teamSitePath": "The relative path to the SharePoint site associated with the team",
+    "channelRelativeUrl": "The relative path to the SharePoint folder associated with the channel",
+    "sessionId": "The unique ID for the current Teams session for use in correlating telemetry data",
+    "userTeamRole": "The user's role in the team",
+    "isTeamArchived": "Indicates if team is archived",
+    "hostClientType": "The type of host client. Possible values are android, ios, web, desktop, rigel",
+    "frameContext": "The context where tab URL is loaded (content, task, setting, remove, sidePanel)",
```
[the remark this answers is not in the document]
Comment (Mahesh2516) [reply msteams-platform/tabs/how-to/access-teams-context.md:96]: added "for example"

File: msteams-platform/tabs/how-to/access-teams-context.md [hunk 2]
```diff
@@ -73,20 +73,37 @@ The context variable will look like the following example.
...
+    "sharepoint": "The SharePoint context is available only when hosted in SharePoint",
+    "tenantSKU": "The type of license for the current user tenant",
+    "userLicenseType": "The license type for a current user",
```
[the remark this answers is not in the document]
Comment (Mahesh2516) [reply msteams-platform/tabs/how-to/access-teams-context.md:99]: Corrected

File: msteams-platform/tabs/how-to/access-teams-context.md [hunk 3]
```diff
@@ -73,20 +73,37 @@ The context variable will look like the following example.
...
+    "teamSitePath": "The relative path to the SharePoint site associated with the team",
+    "channelRelativeUrl": "The relative path to the SharePoint folder associated with the channel",
+    "sessionId": "The unique ID for the current Teams session for use in correlating telemetry data",
+    "userTeamRole": "The user's role in the team",
+    "isTeamArchived": "Indicates if team is archived",
+    "hostClientType": "The type of host client. Possible values are android, ios, web, desktop, rigel",
+    "frameContext": "The context where tab URL is loaded (for example, content, task, setting, remove, sidePanel)",
+    "sharepoint": "The SharePoint context is available only when hosted in SharePoint",
+    "tenantSKU": "The license type for the current user tenant",
+    "userLicenseType": "The license type for a current user",
```
Comment (surbhigupta12) [inline msteams-platform/tabs/how-to/access-teams-context.md:99]: Please fix the article usage.

**Review by surbhigupta12 — COMMENTED** [review]
Minor comment. Please verify.

File: msteams-platform/tabs/how-to/access-teams-context.md [hunk 4]
```diff
@@ -73,20 +73,37 @@ The context variable will look like the following example.
...
     "groupId": "Guid identifying the current O365 Group ID",
     "theme": "The current UI theme: default | dark | contrast",
-    "isFullScreen": "Indicates whether the tab is in full-screen mode",
-    "userLicenseType": "Indicates the user licence type in the given SKU (for example, student or teacher)",
-    "tenantSKU": "Indicates the SKU category of the tenant (for example, EDU)",
-    "channelType": "microsoftTeams.ChannelType.Private | microsoftTeams.ChannelType.Regular"
+    "isFullScreen": "Indicates if the tab is in full-screen",
+    "teamType": "The type of team",
+    "teamSiteUrl": "The root SharePoint site associated with the team",
+    "teamSiteDomain": "The domain of the root SharePoint site associated with the team",
```
[the remark this answers is not in the document]
Comment (Mahesh2516) [reply msteams-platform/tabs/how-to/access-teams-context.md:89]: In this case, it has to be "the".

**Review by KirtiPereira — APPROVED** [review]
Hi @surbhigupta12 - please approve and merge the PR.
Thank you.

**Review by surbhigupta12 — APPROVED** [review]
Thank you for incorporating the comments. Approving the PR.

**Pull request merged**